search-resources-api/src/main/java/no/unit/nva/dynamodb/DataImportHandler.java

```diff
@@ -0,0 +1,93 @@
+package no.unit.nva.dynamodb;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import no.unit.nva.search.ElasticSearchHighLevelRestClient;
+import no.unit.nva.search.exception.ImportException;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.ApiGatewayHandler;
+import nva.commons.handlers.RequestInfo;
+import nva.commons.handlers.RestRequestHandler;
+import nva.commons.utils.Environment;
+import nva.commons.utils.JacocoGenerated;
+import org.apache.http.HttpStatus;
+import org.slf4j.LoggerFactory;
+
+import java.time.Instant;
+import java.util.Objects;
+
+public class DataImportHandler extends ApiGatewayHandler<ImportDataRequest, ImportDataCreateResponse> {
```

Comment:
As for now we start via API-gateway, it could be an event, direct lambda call or running a docker-image... I guess we will have to rewrap the code when we have more data - whick take more than 15 minutes to run..